compiler/vmdef.nim

```diff
@@ -207,6 +207,7 @@ type
     loopIterations*: int
     comesFromHeuristic*: TLineInfo # Heuristic for better macro stack traces
     callbacks*: seq[tuple[key: string, value: VmCallback]]
+    specialOps*: seq[VmCallback]
```

Comment:
You can just register a callback with a name like ``bindSymImpl``, right? Then we wouldn't need yet another seq in ``vmdef``

Reply:
no, bindSym can be called from different modules/scope, it can have different PContext/scope.
if using available VM callback, I cannot update the PContext/scope.

Reply:
with exception we don't use the `key` if I put it  in `callbacks` seq

Reply:
Yeah, Ok, leave that part as it is then.